pkg/daemon/update.go

```diff
@@ -100,17 +104,58 @@ func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) (bool
 	if !reflect.DeepEqual(oldIgn.Networkd, newIgn.Networkd) {
 		glog.Warningf("daemon can't reconcile state!")
 		glog.Warningf("Ignition networkd section contains changes")
-		return false, nil
+		return false, nil, nil
 	}
 
 	// Passwd section
 
 	// we don't currently configure groups or users in place. we can't fix it if
 	// something changed here.
 	if !reflect.DeepEqual(oldIgn.Passwd, newIgn.Passwd) {
-		glog.Warningf("daemon can't reconcile state!")
-		glog.Warningf("Ignition passwd section contains changes")
-		return false, nil
+		// MCD will handle only ignition SSH keys
+		// This section will check (using some tbd magic) to see if the ONLY diff between 2
+		// Passwd configs is an SSHAuthorizedKey change
+		// Any change other than an SSHChanage will return false.
+
+		var onlySSHChanged bool
```

Comment:
nit: It is true that bools are false by default ingolang, but `onlySSHChanged := false` would be a bit more clear for a casual reader.

Reply:
Done.